test/JDBC/expected/babel_table_type.out

```diff
@@ -649,7 +649,114 @@ test_name#!#1
 drop function mstvf_constraints;
 GO
 
+
+--BABEL-3311 TVF name truncation
```

Comment:
Please use upgrade test (latest --> latest) format so that it can be covered in the dump/restore and future mVU/MVU tests, i.e., create functions in the prepare script.

Reply:
Copied the tests into a set of new upgrade scripts